portfolio/src/main/webapp/comments.js

```diff
@@ -25,34 +25,38 @@ function loadComments(){
   });
 }
 
+function deleteComments(){
+  const request = new Request('/delete-data', {method: 'POST'});
+  fetch(request).then(response => {
+    if (response.redirected) {
+        window.location.href = response.url;
+    }
+  });
+}
+
 /** Creates an <li> element containing text. */
 function createCommentElement(email, comment, time) {
   const card = document.createElement("div"); 
   addClass(card, "card");
-  debugLog("created card")
```

Comment:
Nit: this should probably be a separate PR. It's probably more work than it's worth to fix that in this PR, but in the future, try to keep cleanups separate where possible.